specs/latest/2.0/index.html

```diff
@@ -699,12 +695,23 @@ <h2 class="no-toc">Table of contents</h2>
   const GLenum MAX_CLIENT_WAIT_TIMEOUT_WEBGL                 = 0x9247;
 
   /* Buffer objects */
+  // WebGL1:
+  void bufferData(GLenum target, GLsizeiptr size, GLenum usage);
+  void bufferData(GLenum target, BufferDataSource? srcData, GLenum usage);
+  void bufferSubData(GLenum target, GLintptr dstByteOffset, BufferDataSource? srcData);
```

Comment:
Now that the signatures of these are unchanged from WebGL 1.0, it isn't really necessary to repeat them here.